src/o365/spo/commands/tenant/tenant-recyclebinitem-restore.ts

```diff
@@ -51,7 +51,7 @@ class SpoTenantRecycleBinItemRestoreCommand extends SpoCommand {
         this.waitForRestoreDeletedSite(args.options.url, spoAdminUrl, cmd, response);
       }
       else {
-        cmd.log('Site Collection has not been restored');
+        cb('Site Collection has not been restored');
```

Comment:
How do I "test" this line so that it's code covered?

Reply:
The mocked request would need to throw an error (or whatever response leads to this line)